tests/baselines/reference/api/typescript.d.ts

```diff
@@ -14,93 +14,31 @@ and limitations under the License.
-            readonly typeAcquisition: TypeAcquisition;
-            readonly unresolvedImports: SortedReadonlyArray<string>;
-            readonly cachePath?: string;
-            readonly kind: "discover";
-        }
-        interface CloseProject extends TypingInstallerRequestWithProjectName {
-            readonly kind: "closeProject";
-        }
-        interface TypesRegistryRequest {
-            readonly kind: "typesRegistry";
-        }
-        interface InstallPackageRequest extends TypingInstallerRequestWithProjectName {
-            readonly kind: "installPackage";
-            readonly fileName: Path;
-            readonly packageName: string;
-            readonly projectRootPath: Path;
-            readonly id: number;
-        }
-        interface PackageInstalledResponse extends ProjectResponse {
-            readonly kind: ActionPackageInstalled;
-            readonly id: number;
-            readonly success: boolean;
-            readonly message: string;
-        }
-        interface InitializationFailedResponse extends TypingInstallerResponse {
-            readonly kind: EventInitializationFailed;
-            readonly message: string;
-            readonly stack?: string;
-        }
-        interface ProjectResponse extends TypingInstallerResponse {
-            readonly projectName: string;
-        }
-        interface InvalidateCachedTypings extends ProjectResponse {
-            readonly kind: ActionInvalidate;
-        }
-        interface InstallTypes extends ProjectResponse {
-            readonly kind: EventBeginInstallTypes | EventEndInstallTypes;
-            readonly eventId: number;
-            readonly typingsInstallerVersion: string;
-            readonly packagesToInstall: readonly string[];
-        }
-        interface BeginInstallTypes extends InstallTypes {
-            readonly kind: EventBeginInstallTypes;
-        }
-        interface EndInstallTypes extends InstallTypes {
-            readonly kind: EventEndInstallTypes;
-            readonly installSuccess: boolean;
-        }
-        interface InstallTypingHost extends JsTyping.TypingResolutionHost {
-            useCaseSensitiveFileNames: boolean;
-            writeFile(path: string, content: string): void;
-            createDirectory(path: string): void;
-            getCurrentDirectory?(): string;
-        }
-        interface SetTypings extends ProjectResponse {
-            readonly typeAcquisition: TypeAcquisition;
-            readonly compilerOptions: CompilerOptions;
-            readonly typings: string[];
-            readonly unresolvedImports: SortedReadonlyArray<string>;
-            readonly kind: ActionSet;
-        }
-        interface WatchTypingLocations extends ProjectResponse {
-            /** if files is undefined, retain same set of watchers */
-            readonly files: readonly string[] | undefined;
-            readonly kind: ActionWatchTypingLocations;
-        }
+    export namespace server {
```

Comment:
What declaration is outside of the server namespace such that everything in the ts namespace now gets an export modifier?

Reply:
There’s some stuff in the internal version, like I was showing you earlier. But, looking closer at it, the specific locals that are causing this one seem like they’re being duplicated from `ts.tracingEnabled` to `ts`, so I’ll take another look.

Reply:
Yeah, I was just assuming that because this PR is only changing the protocol that other stuff wouldn't change as our current API is fully exported and complete.